gridt/src/app/home/expandable/expandable.component.ts

```diff
@@ -0,0 +1,24 @@
+import { Component, OnInit, Input } from "@angular/core";
+
+@Component({
+  selector: "app-expandable",
+  templateUrl: "./expandable.component.html",
+  styleUrls: ["./expandable.component.scss"],
+})
+export class ExpandableComponent implements OnInit {
+
+  @Input() signalMessage;
+  itemExpanded = false;
+
+  constructor() { }
+
+  ngOnInit() {}
+
+  async toggleFullMessage() {
+    if(this.signalMessage.length > 10) {
```

Comment:
This is rather arbitrary choice. Are you able to find out if the message is being wrapped or not and then use that information to perform the check?